stories/rich-text-editor/__snapshots__/rich-text-editor.stories.storyshot

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Storyshots Patterns/Rich Text Editor Compact Mode 1`] = `
```

Comment:
storyshots have become empty, please fix this. Usually this is due to formatting differences. Please check some of the other components where storyshots are fine and try similar changes.

Reply:
updated storyshot.